lib/plausible/ingestion/acquisition.ex

```diff
@@ -1,10 +1,38 @@
 defmodule Plausible.Ingestion.Acquisition do
   @moduledoc false
   @external_resource "priv/ga4-source-categories.csv"
+  @mapping_overrides [
+    {"fb", "Facebook"},
+    {"ig", "Instagram"},
+    {"perplexity", "Perplexity"},
+    {"linktree", "Linktree"}
+  ]
+  @custom_source_categories [
+    {"hacker news", "SOURCE_CATEGORY_SOCIAL"},
+    {"yahoo!", "SOURCE_CATEGORY_SEARCH"},
+    {"gmail", "SOURCE_CATEGORY_EMAIL"},
+    {"telegram", "SOURCE_CATEGORY_SOCIAL"},
+    {"slack", "SOURCE_CATEGORY_SOCIAL"},
+    {"producthunt", "SOURCE_CATEGORY_SOCIAL"},
+    {"github", "SOURCE_CATEGORY_SOCIAL"},
+    {"steamcommunity.com", "SOURCE_CATEGORY_SOCIAL"},
+    {"statics.teams.cdn.office.net", "SOURCE_CATEGORY_SOCIAL"},
```

Comment:
```suggestion
    {"statics.teams.cdn.office.net", "SOURCE_CATEGORY_SOCIAL"}, # codespell:ignore
```

This should do the trick according to docs: https://github.com/codespell-project/codespell?tab=readme-ov-file#inline-ignore